Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -211,6 +211,30 @@ Vector<String> HackStudioWidget::selected_file_names() const
     return files;
 }
 
+Vector<LexicalPath> HackStudioWidget::selected_file_paths() const
+{
+    Vector<LexicalPath> files;
+    m_project_tree_view->selection().for_each_index([&](const GUI::ModelIndex& index) {
+        String sub_path = index.data().as_string();
+
+        GUI::ModelIndex parent_or_invalid = index.parent();
+
+        while (parent_or_invalid.is_valid()) {
+            StringBuilder path_from_parent;
+            path_from_parent.appendff("{}/{}", parent_or_invalid.data().as_string(), sub_path);
+
+            sub_path = path_from_parent.build();
+
+            parent_or_invalid = parent_or_invalid.parent();
+        }
+        StringView sub_path_sv(sub_path);
```

Comment:
`StringView` can be implicitly constructed from a `String`, so there's no need to create the `sub_path_sv` local variable, you can simply do `LexicalPath path(sub_path)`